tox.ini

```diff
@@ -18,10 +15,6 @@ requires = pip >= 18.0
 [testenv]
 changedir = .tmp/{envname}
 deps =
-    sphinx13: sphinx==1.3.6
-    sphinx14: sphinx==1.4.9
-    sphinx15: sphinx==1.5.6
-    sphinx16: sphinx==1.6.7
     sphinx17: sphinx==1.7.9
     sphinx18: sphinx==1.8.5
     sphinx20: sphinx==2.0.1
```

Comment:
@astrofrog - unrelated to this PR, but why do we pin the version so precisely? This case no newer bugfix releases get tested (surely, it's only relevant for the latest release). What about setting `sphinx==2.1.*` instead? (I couldn't comment on the next line).

Reply:
Oh, does * work for pip?

Reply:
yes, it does, we do that in `ci-helpers`, too :)

Reply:
I didn't want to do this in the commit I just pushed up since I'm not sure I fully understand the syntax here.  So I'd say this could be a follow-on separate PR?